packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -1060,4 +1065,33 @@ class TextPainter {
     assert(_debugAssertTextLayoutIsValid);
     return  _lineMetricsCache ??= _paragraph!.computeLineMetrics();
   }
+
+  bool _disposed = false;
+
+  /// Whether this object has been disposed or not.
+  ///
+  /// Only for use when asserts are enabled.
+  bool get debugDisposed {
+    bool disposed = false;
+    assert(() {
+      disposed = _disposed;
+      return true;
+    }());
+    return disposed;
+  }
+
+  /// Releases the resources associated with this painter.
+  ///
+  /// After disposal, this painter is unusable.
```

Comment:
nit: I think this sentence does not need a comma.

Reply:
Done